### DISCUSSION TITLE
Where to report a problem with waterway routing

### DISCUSSION THREAD

**Discussion by beilharz** [osmandapp/OsmAnd#11603] [Maps Related]
There is some issue with the routing across "Schiffshebewerk Niederfinow" (https://www.openstreetmap.org/way/37112270). Routing from one side to the other results in a 57km route.
My understanding of osm and osmand is too shallow to understand if the problem is in osm data or the interpretation of it by osmand. Any advise?

**Comment by vshcherb** [comment]
the issue is related to https://www.openstreetmap.org/way/25824407 - <select value="1" t="waterway" v="boat_lift"/> was not supported by OsmAnd.
 I've fixed it with https://github.com/osmandapp/OsmAnd-resources/commit/1e0bcb50f35228959f7e1b31f6e972a12c42c5f9.

The fix requires new OsmAnd version + new Maps of this region (could be created with MapCreator) https://docs.osmand.net/en/main@latest/development/map-creation/create-offline-maps-yourself#osmandmapcreator

**Reply by beilharz** [reply]
Thank you for the swift reply and fix! Is there a simple way to see which version (will) include this commit?

**Reply by vshcherb** [reply]
every new nightly build or next release (probably 4.0 name) .
